docs/design-structure-support/technical-assistance.md

```diff
@@ -3,58 +3,58 @@ sidebar_position: 1
 
-Licensing may seem like a technical detail and it is easy to overlook, but the type of license you choose can have wide-reaching and lasting effects on your company, so you must choose wisely. It is also important to decide at this stage whether you would like your license to be copyleft, which ensures that downstream derivatives stay open source; or permissive, which allows downstream derivatives to be redistributed as closed source.  The guide will talk about the options for different licenses to explore: software, hardware and content. 
+Licensing may seem like a technical detail and it is easy to overlook, but the type of license you choose can have wide-reaching and lasting effects on your company, so you must choose wisely. It is also important to decide at this stage whether you would like your license to be copyleft, which ensures that downstream derivatives stay open source; or permissive, which allows downstream derivatives to be redistributed as closed source.  The guide will talk about the options for different licenses to explore: software, hardware and content.
 
-## Conduct a Needs Assessment on Startup Support 
-Before designing or supporting the startup's open source journey and becoming a DPG, you will need to conduct a needs assessment on what support they will need.  This needs assessment includes, but not limited to: 
+## Conduct a needs assessment on startup support
+Before designing or supporting the startup's open source journey and becoming a DPG, you will need to conduct a needs assessment on what support they will need.  This needs assessment includes, but not limited to:
 
-1. Company's Problem Statement 
-2. Company's Solution Statement 
-3. Piloting plan/experiment summary 
-4. Piloting challenges 
-5. Business Model/Scalability after pilot 
-6. Product and User Development – what needs to be changed/fixed for the product to qualify as beta DPG?  
-7. Company performance to date 
-8. Team skills/expertise - what other areas of expertise are missing on the team and why are they crucial for the pilot? 
-9. Business Strategy (business model, user/customer acquisition, potential CO scale/LTA, Potential for public funds, potential for private funds) 
+1. Company's Problem Statement
+2. Company's Solution Statement
+3. Piloting plan/experiment summary
+4. Piloting challenges
+5. Business Model/Scalability after pilot
+6. Product and User Development – what needs to be changed/fixed for the product to qualify as beta DPG?
+7. Company performance to date
+8. Team skills/expertise - what other areas of expertise are missing on the team and why are they crucial for the pilot?
+9. Business Strategy (business model, user/customer acquisition, potential CO scale/LTA, Potential for public funds, potential for private funds)
 10. Technical Support Plan Summary
 
 _**Resource: The detailed [needs assessment template can be found here](https://docs.google.com/document/d/1nZ9YsVc-ENKXzoMP2KEQjHIc-k_X_1-N/edit).**_
```

Comment:
This document is publicly editable -- I'd recommend making it publicly comment-able (or just view-able) for the launch.

Reply:
Will look into this!